test/lib/display.test.ts

```diff
@@ -0,0 +1,254 @@
+import { readFile } from "fs";
+import { join } from "path";
+import { test } from "tap";
+
+import { DepGraphData } from "@snyk/dep-graph";
+import { display } from "../../lib/display";
+import { Options, ScanResult, TestResult } from "../../lib/types";
+
+function readFixture(fixture: string, filename: string): Promise<string> {
+  const dir = join("./", "test", "fixtures", fixture);
+  const file = join(dir, filename);
+  return new Promise((resolve, reject) => {
+    readFile(file, "utf-8", (err, data) => {
+      if (err) {
+        reject(err);
+      }
+      resolve(data);
+    });
+  });
+}
+
+test("display", async (c) => {
```

Comment:
Ah, this is a fantastic candidate for jest snapshot testing that @ivanstanev added! I think if Ivan's PR has to go in first for any reason, it would be worth considering adding :-)

Reply:
Definitely!